webrtc.html

```diff
@@ -1676,8 +1676,8 @@
             <li>Let <var>connection</var> be the target
             <code><a>RTCPeerConnection</a></code> object.</li>
             <li>If <code><var>configuration</var>.peerIdentity</code> is
-            set and its value differs from the <a>target peer
-            identity</a>, <a>throw</a> an <code>InvalidModificationError</code>.
+            set and <a>[[\Configuration]]</a>.<code>peerIdentity</code> is
+            not set, <a>throw</a> an <code>InvalidModificationError</code>.
```

Comment:
This sounds like it allows the peer identity to change, just not go from being unset to being set. I don't think that's what we want. I thought the plan was to give everything a default value (`null` for `peerIdentity`, `[]` for `certificates`, etc.), and just say "if the value is different, throw an `InvalidModificationError`".